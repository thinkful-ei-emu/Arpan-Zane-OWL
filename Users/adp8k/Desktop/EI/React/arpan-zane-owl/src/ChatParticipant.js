import React from 'react';


function ChatParticipants(props){
    return (
        <div>
            <p>{props.type}</p>
            <p>{props.message}</p>
            <p>{props.time}</p>
        </div>
    )
}

export default ChatParticipants;