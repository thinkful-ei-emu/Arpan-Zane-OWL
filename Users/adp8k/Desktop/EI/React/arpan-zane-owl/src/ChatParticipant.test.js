import React from 'react';
import ReactDOM from 'react-dom';
import ChatParticipant from './ChatParticipant'
import renderer from 'react-test-renderer';


describe('<ChatParticipant />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');

        ReactDOM.render(<ChatParticipant type="message" message="Hello" time="1548852646559" />, div);


        ReactDOM.unmountComponentAtNode(div);

    });

    it('renders this UI as expected', () => {
        // Render the component, as JSON
        const tree = renderer.create(<ChatParticipant type="message" message="Hello" time="1548852646559"/>).toJSON();
        // Check whether it matches the previous snapshot
        // Stored in __snapshots__/App.test.js.snap
        expect(tree).toMatchSnapshot(); 
    });
});