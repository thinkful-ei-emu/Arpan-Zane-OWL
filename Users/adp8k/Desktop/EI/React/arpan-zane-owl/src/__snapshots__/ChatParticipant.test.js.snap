// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChatParticipant /> renders this UI as expected 1`] = `
<div>
  <p>
    message
  </p>
  <p>
    Hello
  </p>
  <p>
    1548852646559
  </p>
</div>
`;
